src/models.rs

```diff
@@ -3,22 +3,38 @@
 use crate::{color::Color, get_context};
 
 use crate::{quad_gl::DrawMode, texture::Texture2D};
-use glam::{vec2, vec3, Quat, Vec2, Vec3};
+use glam::{vec2, vec3, vec4, Quat, Vec2, Vec3, Vec4};
 
+#[repr(C)]
 #[derive(Clone, Debug, Copy)]
 pub struct Vertex {
     pub position: Vec3,
     pub uv: Vec2,
-    pub color: Color,
+    pub color: [u8; 4],
```

Comment:
I wish we could use `color: Vec4` or `color: Color`, it would be one extra general purpose attribute for custom shaders. Unfortunately all current shaders expect to receive `color` in 0..255 range and do this `color / 255.0` thing. Lets wait with this for the next major version bump!